### DISCUSSION TITLE
Wrong translation for transaction linking (german)

### DISCUSSION THREAD

**Discussion by DunklerPhoenix** [firefly-iii/firefly-iii#5945] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

While linking transactions you have the options:
refund and reimbursed

In the german translation they are both translated with the same word

### Expected behaviour

Better Translation for all options (except "relates to")

<table>
<tr>
	<td>is (partially) paid for by 
	<td>wird (teilweise) bezahlt durch
<tr>
	<td>(partially) pays for
	<td>zahlt (teilweise) für 
<tr>
	<td>is (partially) refunded by
	<td>wird (teilweise) erstattet durch
<tr>
	<td>(partially) refunds
	<td>erstattet (teilweise)
<tr>
	<td>is (partially) reimbursed by
	<td>wird (teilweise) beglichen durch
<tr>
	<td>(partially) reimburses
	<td>begleicht (teilweise)
</table>

### Debug information

Debug information generated at 2022-03-29 11:48:26 Europe/Zurich for Firefly III version **5.6.16**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.6.16 |
| Firefly III API | 1.5.5 |
| Build | 113, 02-03-2022 19:39:22 CET |
| PHP | 8.0.16 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | f5ec621d-3ef9-4779-8d7c-d8a219fe3354 |
| Using docker? | true |
| Layout | v1 |
| System TZ | Europe/Zurich |
| Browser TZ | Europe/Zurich |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | de_DE |
| Default locale | equal |
| BCscale | 24 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | mysql |
| DB version | 18 (exp. 18) |
| Login provider | eloquent  |
| Trusted proxies (.env) | ** |

| User info | Value |
| --- | --- |
| Session start | 2022-03-01 00:00:00 |
| Session end | 2022-03-31 23:59:59 |
| Session first | 2022-03-24 00:00:00 |
| User ID | 1 |
| User language | de_DE |
| User locale | de_DE |
| Attempt at "de_DE.utf8" | 'de_DE.utf8' |
| Attempt at "de_DE.UTF-8" | 'de_DE.UTF-8' |
| User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.82 Safari/537.36 |

### Additional info

![chrome_Qzb8rbd1Pd](https://user-images.githubusercontent.com/1261305/160582455-27dd94da-8ae0-4f39-b328-74f98faab87a.png)
![chrome_XoNgYDUq9c](https://user-images.githubusercontent.com/1261305/160582461-6f367c7d-662f-4195-9d2e-4833cc474ca0.png)

**Comment by JC5** [comment, marked as answer]
Feel free to submit the new translations on Crowdin. They will be included in the new release:

https://crowdin.com/translate/firefly-iii/142/en-de?filter=basic&value=0#q=paid%20for%20by

**Comment by SmartPhoneLover** [comment]
I would like to ask why the last option is repeated twice: 'relates to'. Is this some kind of bug?
Thank you.

**Reply by JC5** [reply]
No, it's just that the relationship is called "relates to" in both directions.
